ecgtools/builder.py

```diff
@@ -108,6 +108,9 @@ def get_directories(self):
         elif isinstance(self.root_path, list):
             dirs = [x for path in self.root_path for x in path.glob(pattern) if x.is_dir()]
 
+            if not dirs:
+                dirs = self.root_path
```

Comment:
@mgrover1, 

> When building from a list of directories, the current implementation will results in a nested list of the input directories, which causes issues with get_filelist - this ensures that this issue does not happen with a provided list of directories

The code addition in this PR appears to be dealing with a different issue i.e. when `dirs` is empty... Also, code coverage reports that this `if block` is not covered by the tests. I could be missing something though. Do you have a few minutes to chat over a video call?